std_msgs/Header header
float32 fb_delay
Person2DOcclusion[] persons
